### PR TITLE
Move and update `degimpify` script

### PR DESCRIPTION
**Pull request by emabrey**
The images folder had three copies of a bash script called `degimpify`. This PR includes changes to move that script to a singular location as well as improvements to how the bash script was written (including removal of outdated and unnecessary `tempfile` command usage). Additionally, this PR includes a run of this command on the three directories which already contained the `degimpify` script, which modified the `Effects.xpm` Pixmap.

<details>
<summary>Checklist</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

</details>

### CODE REVIEW

**Comment by nbsp** [comment]
Wouldn't this make more sense in a `contrib/` directory? It's a useful helper script, but obviously not part of Tenacity.

**Comment by emabrey** [comment]
> 
> 
> Wouldn't this make more sense in a `contrib/` directory? It's a useful helper script, but obviously not part of Tenacity.

I mean yeah. There are a couple scripts elsewhere too. There are a bunch of them in the locale directory for example. Can we merge these in the images directory? I'll make a separate commit/PR to relocate all these "contributor" scripts to a central location.

**Review by nbsp — APPROVED** [review]
Fine by me, then. Please squash on merge.

File: images/EditButtons/Effects.xpm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* XPM */
-static const char * const Effects_xpm[] = {
+static const char * const Effects[] = {
```
Comment (n0toose) [inline images/EditButtons/Effects.xpm:2]: ?

**Comment by akleja** [comment]
The script changes looks fine by me. However the XPMs vary a bit from place to place. Most notably in the mouse cursors. Perhaps those should be altered at the same time as well so the script would apply to them too?

The mouse cursors string name currently is NameXpm[]. Changing this to Name[] however will create some clashing (or nearly clashing) variable names.
One route to go would be to have the cursor files renamed to NameXpm.xpm (also in the header files ofc)
The other longer route would be to strip the Xpm part and add another prefix or suffix, but that would have to be changed everywhere a cursor is used.

A related topic that might be discussed is the relevance of a lot of these image files. The current theme system (enabled by flag THEME) has been established as the default for quite a while. A lot of these xpm images (for example EditButtons and ControlButtons) are only used when compiling without the THEME flag. Perhaps removing the old unused stuff both from the repo and from the code would be a good idea?

**Comment by emabrey** [comment]
> 
> 
> The script changes looks fine by me. However the XPMs vary a bit from place to place. Most notably in the mouse cursors. Perhaps those should be altered at the same time as well so the script would apply to them too?
> 
> The mouse cursors string name currently is NameXpm[]. Changing this to Name[] however will create some clashing (or nearly clashing) variable names.
> One route to go would be to have the cursor files renamed to NameXpm.xpm (also in the header files ofc)
> The other longer route would be to strip the Xpm part and add another prefix or suffix, but that would have to be changed everywhere a cursor is used.
> 
> A related topic that might be discussed is the relevance of a lot of these image files. The current theme system (enabled by flag THEME) has been established as the default for quite a while. A lot of these xpm images (for example EditButtons and ControlButtons) are only used when compiling without the THEME flag. Perhaps removing the old unused stuff both from the repo and from the code would be a good idea?

The `Effects.xpm` array was already supposed to be named `Effects` to match the rest of the scripts in the directory. I think whoever added that Pixmap forgot to run the previous script. I verified that the build worked without the previous Effects_xpm name, so I don't think it's even used to begin with. I would like to match the rest of the XPMs to the consistent naming standard, so I will look through and make those modifications as a part of this PR.

File: images/EditButtons/Effects.xpm (hunk 1 above)
Comment (emabrey) [reply images/EditButtons/Effects.xpm:2]: I don't understand what a question mark is supposed to indicate to me.

**Comment by akleja** [comment]
> The `Effects.xpm` array was already supposed to be named `Effects` to match the rest of the scripts in the directory. I think whoever added that Pixmap forgot to run the previous script. I verified that the build worked without the previous Effects_xpm name, so I don't think it's even used to begin with. I would like to match the rest of the XPMs to the consistent naming standard, so I will look through and make those modifications as a part of this PR.

After quite some time spent of manually editing xpm files to match differing naming conventions, and often resoritng to making small adjustments in vim instead just to not have to deal with the whole gimp export -> manually editing workflow, this is much appreciated!

File: images/EditButtons/Effects.xpm (hunk 1 above)
Comment (n0toose) [reply images/EditButtons/Effects.xpm:2]: > I don't understand what a question mark is supposed to indicate to me.

I didn't understand why that change was made, but this seems to have been answered later down below.

**Pull request merged**